lib/core/presentation/pages/chat/chat_message/widgets/message_item/html_message_widget.dart

```diff
@@ -377,35 +377,16 @@ class MatrixPill extends StatelessWidget {
   Widget build(BuildContext context) {
     return InkWell(
       // onTap: UrlLauncher(outerContext, uri).launchUrl,
```

Comment:
Remove this comment, or maybe add TODO, or maybe just print to know that we must handle onTap for this